README.md

```diff
@@ -2700,3 +2700,11 @@ I'm glad to join Kottans
 ## Greetings, lady and gentlemen.
 My name is [Viktor](https://github.com/urchnk) and I am a Python dev, although it is never too late to turn into a FullStack.
 Looking forward to great cooperation! Live long and prosper. 🖖
+
+
+
+# Hi everyone!
+My name is Masha and I am very glad that I saw the news about this course.
+I really want to take the course and learn new skills.
+<img src="assets/images/2.jpg"/>
```

Comment:
PR kind of looks quite nice, we just can't have < / > in markdown, check the other comments pls